docs/metrics.md

```diff
@@ -166,39 +166,49 @@ soroban.host-fn-op.read-entry                | meter     | number of entries rea
 soroban.host-fn-op.write-entry               | meter     | number of entries written
 soroban.host-fn-op.read-key-byte             | meter     | number of key bytes in read entries
 soroban.host-fn-op.write-key-byte            | meter     | number of key bytes in written entries
-soroban.host-fn-op.read-ledger-byte          | meter     | number of data + code bytes in read entries
+soroban.host-fn-op.read-ledger-byte          | meter     | number of entry (data + code) bytes in read entries
 soroban.host-fn-op.read-data-byte            | meter     | number of data bytes in read entries
 soroban.host-fn-op.read-code-byte            | meter     | number of code bytes in read entries
-soroban.host-fn-op.write-ledger-byte         | meter     | number of data + code bytes in written entries
+soroban.host-fn-op.write-ledger-byte         | meter     | number of entry (data + code) bytes in written entries
```

Comment:
same wrt writes -- maybe just "number of bytes written for Soroban entries"

Reply:
Fixed.